packages/Card/src/components/Text/Text.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import PropTypes from "prop-types";
+import * as sc from "./Text.styles";
+
+const propTypes = {
+  /** Body content of the Text. */
+  children: PropTypes.node,
+  shouldTruncate: PropTypes.bool,
+};
+
+const defaultProps = {
+  children: null,
+  shouldTruncate: false,
+};
+
+const Text = props => {
+  const { children, shouldTruncate } = props;
+
+  const textProps = {
+    children,
+    shouldTruncate,
+  };
+
+  function truncateText(text, num) {
+    if (text.length <= num) {
+      return text;
+    }
+    return text.slice(0, num).concat("...");
```

Comment:
Not sure if it is but do have the `lodash.truncate` package already available and is being used in our Collapsible text component, may not be necessary though, https://github.com/acl-services/paprika/blob/master/packages/CollapsibleText/src/CollapsibleText.js#L29